sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowToStorageApiProtoTest.java

```diff
@@ -1049,6 +1050,58 @@ public void testIntegerTypeConversion() throws DescriptorValidationException {
     }
   }
 
+  @Test
+  public void testTimestampTypeConversion() throws DescriptorValidationException {
```

Comment:
I think the test needed for this fix is the Timestamp input value bounds test.

Since they were originally designed to properly parse input values of various "formats", I thought it would be a little too much to include a boundary test there as well.🤔

But, yes I can add the test case to the general test instead of testTimestampTypeConversion. Do you think I should add the test case to both of with and without a field named "f"? (= e.g BASE_TABLE_SCHEMA and BASE_TABLE_SCHEMA_NO_F) Or one of them?

Reply:
@ahmedabu98 I have pushed a new commit. 👍

Reply:
sorry got busy with other things
> I think the test needed for this fix is the Timestamp input value bounds test.

 you're right this case is specific to an upper bound, but i think it still fits in a parsing test. thanks for adding it :)